portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,13 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Kenneth Kenfack's Portfolio</h1>
+      <p>This is my portfolio. It's all about me a Cameroonian king. Growing up to be amazing. I'm currently a sophmore at umbc</p>
+      <p> I love going on adventures, trying new things, track and field, video games, basketball </p>
```

Comment:
I think that you should remove the space in `<p> I` and `basketball </p>`. Spaces in conventional HTML flow are a somewhat malleable concept. If you want a consistent indent at the start of a block, I'd recommend the [content](https://www.w3schools.com/CSSref/pr_gen_content.asp) property and then you can set the style of the paragraph by making it a specific class.